pkg/daemon/ceph/client/image.go

```diff
@@ -71,7 +71,9 @@ func CreateImage(context *clusterd.Context, clusterName, name, poolName, dataPoo
 		size = ImageMinSize
 	}
 
-	sizeMB := int(size / 1024 / 1024)
+	// Roundup the size of the storage image, eg, requested 1048698 bytes should be 2MB while not truncated 1MB
+	sizeMB := int((size + ImageMinSize - 1) / ImageMinSize)
```

Comment:
The comment here could be clarified a bit I think, I had to think about the math for longer than I'm proud of.  perhaps include the idea that we only create images on 1 MB boundaries and we should never create an image size that's smaller than the requested size.  Going up to the next MB (bigger than the requested size) is OK though.

can you include a unit test that verifies the expected imageSize based on the requested image size?  Sounds like the following test cases:
* < 1MB -> 1 MB
* 1 MB -> 1 MB
* 1 MB + 1 -> 2 MB
* 2 MB - 1 -> 2 MB
* 2 MB -> 2 MB
* 2 MB +1 -> 3 MB

Reply:
Hi @jbw976 , thanks for the comments! I've updated the PR and added the corresponding unit test case to verify the change, it's should be OK.